rust/compiler_builtins.rs

```diff
@@ -36,6 +36,18 @@ macro_rules! define_panicking_intrinsics(
     }
 );
 
+define_panicking_intrinsics!("`f32` should not be used", {
```

Comment:
@nbdd0121 @bjorn3 We got a few new intrinsics generated from `classify`

Reply:
Looks like someone forgot `#[inline]` on `classify`.